themes/ee/asset/javascript/src/cp/files/picker.js

```diff
@@ -47,7 +47,7 @@
 				}
 			}
 
-			if ($('div[data-file-field-react]').length && typeof(FileField) !== 'undefined') {
```

Comment:
@Yulyaswan I think we had a specific reason why this `!undefined` check was added?

Reply:
It was the wrong decision to fix the previous issue. Because it fixed one issue (2390) but created another.